src/attackmate/executors/common/debugexecutor.py

```diff
@@ -27,4 +27,4 @@ def _exec_cmd(self, command: DebugCommand) -> Result:
         if command.exit:
             ret = 1
 
-        return Result('', ret)
```

Comment:
Maybe we should take care that the debug-executor doesn't change RESULT_STDOUT at all

Reply:
setoutputvar is set to false in the debugexecutor, so this will not happen. added clarification for debug, setvar and regex commands in the docs. I would still return "Debug: ..."  in the Result in case someone using it in a script wants to use that for logging smt or the like